src/main/java/com/libtask/library2/entities/User.java

```diff
@@ -0,0 +1,38 @@
+package com.libtask.library2.entities;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.Hibernate;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Table (name = "users")
+public class User {
+    @Id
+    String id;
+    String firstName;
+    String lastName;
+    String email;
+
+    @Override
+    public boolean equals(Object o) {

```

Comment:
use @EqualAndHashCode instead

Reply:
IDE marks that with "Replace @EqualsAndHashCode with recommended equals() and hashCode() implementations." notification